### PR TITLE
Joomla 4 Compatability

### PR DESCRIPTION
**Pull request by ankush-maherwal**

### CODE REVIEW

File: src/com_tjvendors/admin/views/vendor/tmpl/edit_bs5.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+								echo $this->form->renderField('region');
+								echo $this->form->renderField('city');
+								echo $this->form->renderField('other_city');
+								echo $this->form->renderField('zip');
+								echo $this->form->renderField('website_address');
+								echo $this->form->renderField('vat_number');?>
+							</fieldset>
+						</div>
+					</div>
+			<?php echo HTMLHelper::_('bootstrap.endTab');
```
Comment (manojLondhe) [inline src/com_tjvendors/admin/views/vendor/tmpl/edit_bs5.php:132]: fix indentation in all layout files

File: src/com_tjvendors/admin/views/vendor/tmpl/edit_bs2.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+								echo $this->form->renderField('city');
+								echo $this->form->renderField('other_city');
+								echo $this->form->renderField('zip');
+								echo $this->form->renderField('website_address');
+								echo $this->form->renderField('vat_number');?>
+							</fieldset>
+						</div>
+					</div>
+			<?php echo HTMLHelper::_('bootstrap.endTab');
+				echo HTMLHelper::_('bootstrap.addTab', 'myTab', 'name', Text::_('COM_TJVENDORS_TITLE_PAYMENT_DETAILS'));
```
Comment (manojLondhe) [inline src/com_tjvendors/admin/views/vendor/tmpl/edit_bs2.php]: fix indentation in all layout files

File: src/com_tjvendors/admin/views/vendors/tmpl/default_bs5.php [hunk 3]
```diff
@@ -33,12 +34,12 @@
...
 $canOrder  = $user->authorise('core.edit.state', 'com_tjvendors');
-$saveOrder = $listOrder == 'a.`ordering`';
+$saveOrder = $listOrder == 'v.ordering';
 
 if ($saveOrder)
 {
-	$saveOrderingUrl = 'index.php?option=com_tjvendors&task=vendors.saveOrderAjax&tmpl=component';
-	HTMLHelper::_('sortablelist.sortable', 'vendorList', 'adminForm', strtolower($listDirn), $saveOrderingUrl);
+	$saveOrderingUrl = 'index.php?option=com_tjvendors&task=vendors.saveOrderAjax&tmpl=component' . Session::getFormToken() . '=1';
+	HTMLHelper::_('draggablelist.draggable');
```
Comment (manojLondhe) [inline src/com_tjvendors/admin/views/vendors/tmpl/default_bs5.php:42]: document this somewhere

File: src/com_tjvendors/site/views/vendor/tmpl/edit_bs3.php [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+Factory::getDocument()->addScriptDeclaration(implode("\n", $script));
+?>
+<div id="tjv-wrapper" class="<?php echo COM_TJVENDORS_WRAPPAER_CLASS;?>">
+<?php
+if (Factory::getUser()->id)
+{
+	?>
+	<h1>
+		<?php
+			if ($this->vendor_id)
```
Comment (manojLondhe) [inline src/com_tjvendors/site/views/vendor/tmpl/edit_bs3.php]: fix indents in all files

File: src/com_tjvendors/site/views/vendor/tmpl/edit_bs5.php [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+Factory::getDocument()->addScriptDeclaration(implode("\n", $script));
+?>
+<div id="tjv-wrapper" class="<?php echo COM_TJVENDORS_WRAPPAER_CLASS;?>">
+<?php
+if (Factory::getUser()->id)
+{
+	?>
+	<h1>
+		<?php
```
Comment (manojLondhe) [inline src/com_tjvendors/site/views/vendor/tmpl/edit_bs5.php:48]: fix indentation in all layout files

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: src/com_tjvendors/admin/views/vendor/tmpl/edit_bs2.php (hunk 2 above)
Comment (ankush-maherwal) [reply src/com_tjvendors/admin/views/vendor/tmpl/edit_bs2.php]: done

File: src/com_tjvendors/admin/views/vendor/tmpl/edit_bs5.php (hunk 1 above)
Comment (ankush-maherwal) [reply src/com_tjvendors/admin/views/vendor/tmpl/edit_bs5.php:132]: done

File: src/com_tjvendors/site/views/vendor/tmpl/edit_bs3.php (hunk 4 above)
Comment (ankush-maherwal) [reply src/com_tjvendors/site/views/vendor/tmpl/edit_bs3.php]: done

File: src/com_tjvendors/admin/views/vendors/tmpl/default_bs5.php (hunk 3 above)
Comment (ankush-maherwal) [reply src/com_tjvendors/admin/views/vendors/tmpl/default_bs5.php:42]: done

File: src/com_tjvendors/site/views/vendor/tmpl/edit_bs5.php (hunk 5 above)
Comment (ankush-maherwal) [reply src/com_tjvendors/site/views/vendor/tmpl/edit_bs5.php:48]: done

File: src/com_tjvendors/tjvendors.xml [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
-	<creationDate>26th Jul 2021</creationDate>
-	<copyright>Copyright (C) 2016 - 2021 Techjoomla. All rights reserved.</copyright>
+	<creationDate>25th Jan 2022</creationDate>
+	<copyright>Copyright (C) 2016 - 2022 Techjoomla. All rights reserved.</copyright>
 	<license>http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL</license>
 	<author>Techjoomla</author>
 	<authorEmail>extensions@techjoomla.com</authorEmail>
 	<authorUrl>https://techjoomla.com</authorUrl>
-	<version>1.4.3</version>
+	<version>2.0.0</version>
```
Comment (manojLondhe) [inline src/com_tjvendors/tjvendors.xml:10]: @ankush-maherwal is sql file name changed to match version?

Comment (ankush-maherwal) [reply src/com_tjvendors/tjvendors.xml:10]: yes, its already updated

**Pull request merged**